tiledb/api/src/query/strategy.rs

```diff
@@ -1026,6 +1026,36 @@ impl Cells {
 
         self.filter(&preserve)
     }
+
+    /// Returns a copy of `self` with only the fields in `fields`,
+    /// or `None` if not all the requested fields are present.
+    pub fn projection(&self, fields: &[&str]) -> Option<Cells> {
```

Comment:
Can you point me to usages of those two new functions?

Reply:
This is maybe an example of where splitting the repositories `tiledb-rs` and `tiledb-tables` is sort of inconvenient.  At some point I was using this function in my development branch of `tables`, but now I've realized I no longer need it, and yet here it remains.

Note that the entirety of this file (and anything you see in any `strategy.rs`) amounts to testing code.

As you may see from the other commit I just pushed to this branch, sometimes things that "obviously work" don't... so I'll add some tests for this function

Reply:
Are you saying that you plan to add coverage in a follow up pr?

Reply:
I pushed a commit to this branch

https://github.com/TileDB-Inc/tiledb-rs/pull/142/commits/e489647e8fc72798ef67516beb4d7878c30687c3

Reply:
I do not entirely understand this, I do not want to hold this cr.